src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -652,4 +652,18 @@
   <data name="CloseTabsAfterDefaultCommandKey" xml:space="preserve">
     <value>Close all tabs after the current tab</value>
   </data>
+  <data name="InvalidLinkContent" xml:space="preserve">
+    <value>The URI "{0}" is invalid. </value>
+    <comment>{0} will be replaced by the URI the user is attempting to open. </comment>
+  </data>
+  <data name="InvalidLinkDialog.PrimaryButtonText" xml:space="preserve">
+    <value>Cancel</value>
+  </data>
+  <data name="UnsupportedSchemeContent" xml:space="preserve">
+    <value>The scheme "{0}" is not supported. </value>

```

Comment:
May be a little too technical. @cinnamon-msft can you suggest a wording? The sentiment we are trying to convey is "You clicked a link that we don't think we can trust; it is a `file` link or a `mailto` link or a `ms-settings` link"

Reply:
"This link type is currently not supported." ?

Reply:
ah, i like that. if we do that, we should include the whole URI (possibly on another line)

Reply:
_and_ if we do that we may want to put it in a textblock whose font is . . . cascadia mono!